server/src/main/java/org/eclipse/openvsx/publish/PublishExtensionVersionHandler.java

```diff
@@ -53,22 +60,23 @@ public class PublishExtensionVersionHandler {
     ExtensionValidator validator;
 
     @Transactional
-    public ExtensionVersion createExtensionVersion(ExtensionProcessor processor, PersonalAccessToken token) {
+    public ExtensionVersion createExtensionVersion(ExtensionProcessor processor, PersonalAccessToken token, LocalDateTime mirror) {
```

Comment:
Can you change `LocalDateTime mirror` to `boolean checkDependencies`?

Reply:
@amvanbaren is this still relevant? If so, I'm not sure where to use and get this new argument 😄